src/polyswarm_api/__main__.py

```diff
@@ -214,31 +215,52 @@ def url_scan(ctx, url, url_file, force, timeout):
     ctx.obj['output'].write(str(rf))
 
 
-@click.option('-r', '--hash-file', help='File of hashes, one per line.', type=click.File('r'))
-@click.option('--hash-type', help='Hash type to search [sha256|sha1|md5], default=sha256', default='sha256')
-@click.argument('hash', nargs=-1, callback=validate_hash)
-@polyswarm.command('search', short_help='search for hash')
+@click.option('-r', '--hash-file', help="File of hashes, one per line.", type=click.File('r'))
```

Comment:
Standardize to single quotes in this file.

Reply:
Oops. I missed one. Thanks!